library/common-library/DEVELOPERS.md

```diff
@@ -0,0 +1,615 @@
+{{- /* For the agent */}}
+{{ include "newrelic.common.images.image" ( dict "imageRoot" .Values.images.agent "context" .) }}
+```
+
+### `newrelic.common.images.registry`
+It returns the registry from the global or local values. You should avoid using this helper to create your image
+URL and use `newrelic.common.images.image` instead, but it is there to be used in case it is needed.
+
+Usage:
+```mustache
+{{- /* For jobImage */}}
+{{ include "newrelic.common.images.registry" ( dict "imageRoot" .Values.jobImage "context" .) }}
+{{- /* For the integration */}}
+{{ include "newrelic.common.images.registry" ( dict "imageRoot" .Values.images.integration "context" .) }}
+{{- /* For the agent */}}
+{{ include "newrelic.common.images.registry" ( dict "imageRoot" .Values.images.agent "context" .) }}
+```
+
+### `newrelic.common.images.repository`
+It returns the image from the values. You should avoid using this helper to create your image
+URL and use `newrelic.common.images.image` instead, but it is there to be used in case it is needed.
+
+Usage:
+```mustache
+{{- /* For jobImage */}}
+{{ include "newrelic.common.images.repository" ( dict "imageRoot" .Values.jobImage "context" .) }}
+{{- /* For the integration */}}
+{{ include "newrelic.common.images.repository" ( dict "imageRoot" .Values.images.integration "context" .) }}
+{{- /* For the agent */}}
+{{ include "newrelic.common.images.repository" ( dict "imageRoot" .Values.images.agent "context" .) }}
+```
+
+### `newrelic.common.images.tag`
+It returns the image's tag from the values. You should avoid using this helper to create your image
+URL and use `newrelic.common.images.image` instead, but it is there to be used in case it is needed.
+
+Usage:
+```mustache
+{{- /* For jobImage */}}
+{{ include "newrelic.common.images.tag" ( dict "imageRoot" .Values.jobImage "context" .) }}
+{{- /* For the integration */}}
+{{ include "newrelic.common.images.tag" ( dict "imageRoot" .Values.images.integration "context" .) }}
+{{- /* For the agent */}}
+{{ include "newrelic.common.images.tag" ( dict "imageRoot" .Values.images.agent "context" .) }}
+```
+
+### `newrelic.common.images.renderPullSecrets`
+If returns a merged map that contains the pull secrets from the global configuration and the local one.  
+
+Usage:
+```mustache
+{{- /* For jobImage */}}
+{{ include "newrelic.common.images.renderPullSecrets" ( dict "pullSecrets" .Values.jobImage.pullSecrets "context" .) }}
+{{- /* For the integration */}}
+{{ include "newrelic.common.images.renderPullSecrets" ( dict "pullSecrets" .Values.images.pullSecrets "context" .) }}
+{{- /* For the agent */}}
+{{ include "newrelic.common.images.renderPullSecrets" ( dict "pullSecrets" .Values.images.pullSecrets "context" .) }}
+```
+
+
+
+## _serviceaccount.tpl
+These functions are used to evaluate if the service account should be created, with which name and add annotations to it.
+
+The functions that the common library has implemented for serve account are:
```

Comment:
```suggestion
The functions that the common library has implemented for service account are:
```

Reply:
Fixed in [#6216d9f](https://github.com/newrelic/helm-charts/pull/797/commits/6216d9f3f09fedf3717c1b7a457a887fd8230371)